### DISCUSSION TITLE
Better Folders

### DISCUSSION THREAD

**Discussion by AlwaysAlpha1** [Zerthox/BetterDiscord-Plugins#192] [General]
It seem with the recent update when you click on your folder it doesnt give you an option to select Custom Icon anymore only the default color, is this able to be fixed? my one computer I was still able to select it, that was until I restarted discord and it also updated. please forgive me if this isnt the right spot to get help or support or even to just report this issue as im not to fimilar with this site to well.

**Comment by Zerthox** [comment]
You can submit bugs like this as [issue](../issues). Will be fixed whenever the update is approved for the BetterDiscord site.
